RELEASE.md

```diff
@@ -4,6 +4,8 @@
 
 ## Bug fixes and other changes
 
+* Updated MatplotlibWriter Dataset docs with working examples.
```

Comment:
The next upcoming release is `0.18.4`, could you move this sentence to that section?

Reply:
Done!